incubator/hnc/internal/validators/object.go

```diff
@@ -77,6 +71,12 @@ func (o *Object) handle(ctx context.Context, log logr.Logger, inst *unstructured
 		return allow("")
 	}
 
+	// Allow changes to the types that are not in propagate mode.
+	if !o.isPropagateType(inst) {
+		log.V(1).Info("Allowed change to non-propagate-mode types.")
```

Comment:
I'd skip this myself, or make it a V(2). I want this to be super-fast and super-safe (e.g. the fewer functions we call, the better).

Reply:
sg. removed.